client.ps1

```diff
@@ -4,6 +4,16 @@ $ErrorActionPreference = "Stop"
 . "$PSScriptRoot\common.ps1"
 . "$PSScriptRoot\extra_software.ps1"
 
+function Configure-TimeSync {
+    Write-Output "Configuring Client VM to synchronize time with Studio VM..."
+    Execute-Command -Path "w32tm.exe" -Arguments "/config /manualpeerlist:$STUDIOIP /syncfromflags:manual /reliable:NO /largephaseoffset:0 /update"
+
+    Set-Service -Name w32time -StartupType Automatic
+    Restart-Service w32time
+
+    Execute-Command -Path "w32tm.exe" -Arguments "/resync /nowait"

```

Comment:
Don't synchronize time here. The studio is not up yet and I don't think we need time synchronized during installation anyway.